src/node-loaders.js

```diff
@@ -31,6 +31,7 @@ var FileSystemLoader = Loader.extend({
                     var watcher = chokidar.watch(p, { ignoreInitial: true });
 
                     watcher.on('all', function(event, fullname) {
+                        fullname = path.resolve(fullname);
```

Comment:
Is `fullname` (from chokidar events) relative to `p` or absolute ?

Because if the path is relative, this will work only if `p == process.cwd()` which is not always the case.